### DISCUSSION TITLE
Feature Request: Search bar for the "Repositories" page

### DISCUSSION THREAD

**Discussion by saagarjha** [community/community#50] [Mobile] [answered]
I have a lot of repositories and they're listed in semi-random order on this page, so it's a bit hard to find what I'm looking for. A search field would make the list a lot more manageable :)

**Comment by candyho** [comment, marked as answer]
Hey thanks for sharing your feedback. Totally get your pain on this one. We're actively looking into improving different in-app search scenarios, will keep you posted on the updates!

**Reply by romulocintra** [reply]
Really needed feature, just jumping. From app and mobile web to have this goodness

**Reply by a4vg** [reply]
Sorting the repositories in alphabetical order would also be a great start.

**Reply by kapitoshka438** [reply]
Thumbs up. I have so many repos and it's difficult to find in the list.

**Reply by koalazub** [reply]
Just checking in on how this is going 1 year later? We've got search for general repos, but not for orgs?

**Reply by D-Pow** [reply]
Related: [#7215](https://github.com/github/feedback/discussions/7215).

It would be nice to be able to search within:

* A repository.
* An organization.
* A file.
* Definitions/References (i.e. the popup that you get in web when clicking on a function).

I didn't realize this was originally opened over a year ago. Has there not been any progress at all with search functionality?
